ledger/src/blockstore_meta.rs

```diff
@@ -11,6 +12,14 @@ use {
     },
 };
 
+bitflags! {
+    #[derive(Default, Serialize, Deserialize)]
+    pub struct ConnectedFlags:u8 {
+        const CONNECTED        = 0b0000_0001;
+        const PARENT_CONNECTED = 0b1000_0000;
```

Comment:
Let's add a comment block stating the definition of `CONNECTED` and `PARENT_CONNECTED`.

I think the CONNECTED now means the current slot is full and all slots from the current slot up to the root are also connected.  Am I right at this part?

Reply:
Yes, connected means this slot is a descendant of genesis, or a snapshot root. Added comments.